doc/dev/design/ui_goals.rst

```diff
@@ -0,0 +1,54 @@
+Ceph Dashboard Design Goals
+===========================
+
+.. note:: this document is intended to provide a focal point for discussing the overall design principles for mgr/dashboard
+
+Introduction
+############
+
+Most distributed storage architectures are inherently complex, and can present a management challenge to Operations teams who are
+typically stretched across multiple product and platform disciplines. In general terms, the complexity of any solution can have a 
+direct bearing on the operational costs incurred to manage it. The answer is simple...make it simple :)
+
+This document is intended to highlight Ceph Dashboard design goals which may help to  
+
+* reduce complexity
+* increase productivity
+* improve time-to-value
+
+ 
+Understanding the Persona of the Target User
+############################################
+
+Ceph has historically been administered from the CLI. The CLI has always and will always offer the richest, most flexible way to 
+install and manage a Ceph cluster. Administrators who require and demand this level of control are unlikely to adopt a UI for anything
+more than a technical curiousity.
+
+The relevance of the UI is therefore more critical for a new SysAdmin, where it can help technology adoption and reduce the operational friction
+that is normally experienced when implementing a new solution.
+
+Understanding the target user persona is therefore a fundamental first step in design. Attempting to design a UI that meets the requirements of a 'seasoned'
+Ceph Administrator or Developer, and a relatively new SysAdmin is unlikely to satisfy either user group.
+
+Design Principles
+#################
+
+Key Principles
+--------------
+
+
+1. **Clarity and consistency**. The UI should ensure the data shown is unambiguous and consistent across different views
```

Comment:
I'd include something like increasing "contextuality" (if such a thing exists): displaying information & directions together with closely related information. Workflows will increase this, but regular monitoring & alerting should also be cross-integrated/linked. For example, basically all asynchronous (non-user driven) events should be linked to actionable/readable Dashboard components (e.g.: Prometheus Alertmanager alerts should point to specific Dashboard components, Dashboard pop-up errors should lead the user to the faulty component, ...)

Reply:
Great point - "contextual design". I'll add this in to try and capture that.